zookeeper-server/src/main/java/org/apache/zookeeper/server/NIOServerCnxn.java

```diff
@@ -81,6 +81,11 @@
      */
     private long sessionId;
 
+    /**
+     * Client socket option for TCP keepalive
+     */
+    private final boolean clientTcpKeepAlive = Boolean.getBoolean("zookeeper.clientTcpKeepAlive");
+
```

Comment:
please doc this new property in the `zookeeperAdmin.md`

Reply:
Thanks your comments. Accept it. I add items in zookeeperAdmin in latest patch. Please review it.